src/containers/OrdinalsArea.js

```diff
@@ -244,6 +249,23 @@ const OrdinalsArea = ({ className, space }) => {
                             )}
                         </button>
                     </div>
+                    <div className="col-lg-1 col-md-1 col-sm-1 col-2">
+                        <button
+                            type="button"
+                            className={clsx(
+                                "sort-button d-flex flex-row justify-content-center",
+                                activeSort === "inscriptionNumber" && "active"
+                            )}
+                            onClick={() => {
+                                sortBy("inscriptionNumber");
+                            }}
```

Comment:
This can cause memory leak . You should extract this into separate  function.

Reply:
@StefanBraunovic I definitely am experiencing the memory leak now that you mention it, but struggling to patch it. Mind explaining a bit more? I already have "sortBy" as a separate function

Apologies if the question is remedial